src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketMirroredRepositoryDescriptorClient.java

```diff
@@ -0,0 +1,7 @@
+package com.atlassian.bitbucket.jenkins.internal.client;
+
+import com.atlassian.bitbucket.jenkins.internal.model.BitbucketMirroredRepositoryDescriptor;
+import com.atlassian.bitbucket.jenkins.internal.model.BitbucketPage;
+
+public interface BitbucketMirroredRepositoryDescriptorClient extends BitbucketClient<BitbucketPage<BitbucketMirroredRepositoryDescriptor>> {
```

Comment:
Also, can we name it something different and not use `Descriptor` what about `BitbucketMirroredRepositoryDetailsClient` ?

Reply:
It to keep it consistent with what it returns `BitbucketMirroredRepositoryDescriptor`

Reply:
As per my other comment, I think we should rename the return type too :)

Reply:
Sorry for not being clear, It's the return type of Bitbucket REST endpoint. Currently the naming for all other Bitbucket entities are consistent with Bitbucket's naming and I would like to keep it the same.
Every thing in `com.atlassian.bitbucket.jenkins.internal.model` package is currently following `Bitbucket<Bitbucket object name>` pattern

Reply:
I'm not thrilled, because `descriptor` is used so much in Jenkins, and means something totally different.. I see where you're coming from though